src/components/tags.js

```diff
@@ -9,6 +9,7 @@ export const hiddenTags = {
 	has_cal: '',
 	'has cal': '',
 	hasnoattachment: '',
+	Notjunk: '',
```

Comment:
```suggestion
	notjunk: '',
```

Because of this https://github.com/nextcloud/mail/blob/4501d9b59a78494412634227cd7c8b635e0ce018/src/components/TagModal.vue#L104C104-L104C115